packages/components/src/index.ts

```diff
@@ -71,11 +71,11 @@ export type { RenderThumbFunction } from './pricing-slider/types';
 // Logos
 export { default as ClientLogoList } from './client-logo-list';
 export { JetpackLogo } from './logos/jetpack-logo';
-export { default as CloudLogo } from './logos/cloud-logo';
-export { default as VIPLogo } from './logos/vip-logo';
-export { default as WooLogo } from './logos/woo-logo';
+export { CloudLogo } from './logos/cloud-logo';
+export { VIPLogo } from './logos/vip-logo';
+export { WooLogo } from './logos/woo-logo';
 export { WordPressLogo } from './logos/wordpress-logo';
-export { default as WooCommerceWooLogo } from './logos/woocommerce-woo-logo';
+export { WooCommerceWooLogo } from './logos/woocommerce-woo-logo';
```

Comment:
One of those two logo components looks unnecessary. 

![Screenshot 2025-02-07 at 15 30 06](https://github.com/user-attachments/assets/47474970-12b9-4b64-b81c-05d909cf5d39)

This is a good idea for a follow-up PR IMHO.